kolibri_explore_plugin/assets/src/constants.js

```diff
@@ -22,3 +22,10 @@ export const pageNameToModuleMap = {
 };
 
 export const CUSTOM_PRESENTATION_TITLE = 'custom-channel-ui';
+
+// List of channel ids and the app name with the corresponding
+// custom-channel-presentation zip bundle
+export const CustomChannelApps = {
+  '1306b5d7d1ce4b98b0039324f368ce6a': 'pbskids-ui',
+  '8b51ac7fc47344a699eaf57c4fd31c70': 'sikana-ui',
```

Comment:
If you accept the suggestion above, 'pbskids-ui' should be 'pbskids-ui.zip' here. Same for `sikana-ui.zip`.